hello-world.md

```diff
@@ -6,11 +6,11 @@
 
 ## Specification
 
-The `Hello World!` program will greet me, the caller.
+Write a `Hello World!` function that takes a single input, the name of someone to greet, and returns an appropriate greeting.
```

Comment:
"that takes a single input" 

The Ruby and I believe other versions also allow calling the method without an argument.